ansible/group_vars/osdsdock.yml

```diff
@@ -23,11 +23,17 @@ dummy:
 ###########
 
 # Change it according to your backend, currently support 'lvm', 'ceph', 'cinder'
-enabled_backend: lvm
+enabled_backend: ceph
```

Comment:
For easier deployment for test, the `lvm` backend is SUGGESTED for default option.